test/server/TypeDBClusterRunner.java

```diff
@@ -45,15 +44,15 @@
     private final Map<String, String> remainingServerOpts;
 
     public static TypeDBClusterRunner create() throws InterruptedException, TimeoutException, IOException {
-        int port = ThreadLocalRandom.current().nextInt(40000, 60000);
-        Ports server = new Ports(port, port+1, port+2);
+        List<Integer> ports = findUnusedPorts(3);
+        Ports server = new Ports(ports.get(0), ports.get(1), ports.get(2));
```

Comment:
find 3 currently unused ports, instead of randomly generating a single, static port that may be in-use and we rely on probability to succeed